controllers/storagecluster/reconcile.go

```diff
@@ -526,6 +531,22 @@ func (obj *ocsCephConfig) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.S
 		return nil
 	}
 
+	// configure public network if the cluster is dualstack, but not multus
+	if sc.Spec.Network != nil && sc.Spec.Network.Provider == "" && sc.Spec.Network.DualStack {
+		networkConfig := &configv1.Network{ObjectMeta: metav1.ObjectMeta{Name: "cluster"}}
```

Comment:
@rohantmp shouldnt we do a get call to get the network object here?

Reply:
Yup! Don't know how I missed that!